db/warehouse/migrate/20200417164547_add_youth_families.rb

```diff
@@ -0,0 +1,45 @@
+class AddYouthFamilies < ActiveRecord::Migration[5.2]
+  def change
+    add_column :nightly_census_by_project_clients, :youth_families, :jsonb, default: []
```

Comment:
Use `change_table` here?

Reply:
I guess it would be more DRY.  I tend to be less particular about migrations since they are generally fairly short and simple.